tigerpath/majors_and_certificates/degrees/BSE.json

```
@@ -250,18 +250,33 @@
           "min_needed": 4,
           "explanation": "The humanities and social science courses must include one course in four of the six areas listed below:\n\n- Epistemology and Cognition (EC)\n- Ethical Thought and Moral Values (EM)\n- Foreign Language (at the 107/108 level or above)\n- Historical Analysis (HA)\n- Literature and the Arts (LA)\n- Social Analysis (SA)",
```

Removing the duplicate list in the explanation. It's different for the class of 2024 onwards and it's also a bit redundant anyway, since users can see what the list is by looking a the subrequirements.
```suggestion
          "explanation": "The humanities and social science courses must include one course in four of the six areas listed below.",
```